src/js/control-bar/progress-control.js

```diff
@@ -1,3 +1,7 @@
+import Component from '../component';
+import Slider, { SliderHandle } from '../slider';
```

Comment:
Agreed, I think we should move things like this to their own folders and split apart into multiple files.

#2010